apps/lcs/index.js

```diff
@@ -5,6 +5,7 @@ const clearSession = require('./behaviours/clear-session');
 const sendNotification = require('./behaviours/submit-notify');
 const dateBefore1989 = config.dateBefore1989;
 const checkValidation = require('./behaviours/date-validation.js');
+const redirect = require('./behaviours/custom-redirect');
```

Comment:
I would suggest keeping the name, so it's more explanatory

`const customRedirect = require('./behaviours/custom-redirect');`